sql/dev_fixtures_site.sql

```diff
@@ -1,8 +1,8 @@
 -- sample data for development of the LibreCores web site
 
-INSERT INTO `Organization` (`id`, `name`, `displayName`, `description`, `ownerId`) VALUES
-(1, 'unassigned', 'Unassigned', 'Unassigned description', NULL),
-(2, 'openrisc', 'OpenRISC', 'OpenRISC Description', NULL);
+INSERT INTO `Organization` (`id`, `name`, `displayName`, `description`, `creatorId`, `createdAt`, `updatedAt`) VALUES
+(1, 'unassigned', 'Unassigned', 'Unassigned description', NULL, NULL, NULL),
+(2, 'openrisc', 'OpenRISC', 'OpenRISC Description', NULL, NULL, NULL);
```

Comment:
Agreed, we must place a proper date/time here if these become not null. And proper fixtures are on the way (TM)! :)